third_party/gpus/rocm_configure.bzl

```diff
@@ -214,6 +214,8 @@ def _amdgpu_targets(repository_ctx, rocm_toolkit_path, bash_bin):
         cmd = "%s/bin/rocm_agent_enumerator" % rocm_toolkit_path
         result = execute(repository_ctx, [bash_bin, "-c", cmd])
         targets = [target for target in result.stdout.strip().split("\n") if target != "gfx000"]
+        targets = {x : None for x in targets}
+        targets = list(targets.keys())
```

Comment:
how is this used?

Reply:
I've seen one case where the list of targets returned by rocm_agent_enumerator was "gfx90a,gfx90a,gfx90a,gfx90a,gfx90a,gfx90a". That did not go well with clang. This change removes any duplicates from the list of targets.